common/.eslintrc.fast.json

```diff
@@ -45,6 +57,82 @@
     ],
     "lines-between-class-members": ["error", "always", { "exceptAfterSingleLine": true }],
     "indent": "off",
-    "@typescript-eslint/indent": ["error", 2, { "ignoreComments": true, "flatTernaryExpressions": true }]
+    "@typescript-eslint/indent": ["error", 2, { "ignoreComments": true, "flatTernaryExpressions": true }],
+    "guard-for-in": "error",
+    "new-parens": "error",
+    "no-bitwise": "error",
```

Comment:
I don't like this rule.  I think @bergundy doesn't, either.

Were it more nuanced I might change my mind.

Reply:
I guess it's most useful to catch non-short circuiting int32 `&` and `|`. It's possible to allow all others with `"allow": ["<<", "^", "~"...`, or disable the rule completely and be slightly more careful in codereviews? (Especially when using typescript I think danger is quite low).